pkg/operator/invalidcertscontroller/invalid_certs_controller.go

```diff
@@ -0,0 +1,145 @@
+package invalidcertscontroller
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/prometheus/common/model"
+	"io/ioutil"
+	"k8s.io/client-go/transport"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+
+	prometheusapi "github.com/prometheus/client_golang/api"
+	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
```

Comment:
Nit: if you want to keep imports ordered: builtin, 3-rd party, k8s, openshift :wink:

Reply:
fixed